src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,17 @@
 package mate.academy.service;
 
+import java.util.Optional;
+import mate.academy.model.User;
+
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        Optional<User> byEmail = userService.findByEmail(email);
+        Optional<User> byPassword = userService.findByPassword(password);
+        if (byEmail.isEmpty() || byPassword.isEmpty()) {
+            return false;
+        }
+        return email.equals(byEmail.get().getEmail())
+                && password.equals(byPassword.get().getPassword());
```

Comment:
your logic is not correct
you may obtain login from one user and password from another and consider this combination valid
please find user by email only and check if input password equals with this user's password